__tests__/generate.spec.ts

```diff
@@ -1,7 +1,7 @@
 import * as fs from 'fs';
 import * as fg from 'fast-glob';
 import { mockProcessExit } from 'jest-mock-process';
-import path from 'path/posix';
+import { posix as pathPosix } from 'path';
```

Comment:
why not name it `{posix as path}` instead?
```suggestion
import { posix as path } from 'path';
```

and avoid the rename everywhere?

Reply:
Done ✅